stories/2-elements/buttons.stories.js

```diff
@@ -5,6 +5,25 @@ import omit from 'storybook-react-omit'
 import Button from '../../packages/components/lib/Button'
 import bp from '../../packages/components/lib/breakpoints'
 
+const primaryInfoText = `
+  # BUTTONS
+  Buttons are a very common component used throughout CG interfaces. Almost all calls-to-action are accompanied by one or more buttons. Because of their frequent use, there are several styles to choose from and they should be selected with functionality in mind.
+
+  ## BUTTON RULES:
+    * When grouped, buttons should always be the same size.
+    * When grouped, buttons should not share the same styles (except for size) [ie. a solid red button should not be next to another solid red button]. In these cases, use a different style such as outline and/or a different color combo. Using solid buttons alongside outline buttons of the same color is a safe practice.
+    * Do not manipulate the hover states of buttons, as this can cause inconsistency in the design system.
+
+  ### SOLID
+    Solid buttons are the most commonly used and should be the primary button type used, especially when you have only one button used in a view.
+
+  ### OUTLINE
+    Outline buttons allow for a different visual effect. This is helpful when buttons are shown in groups to break up the monotony that would be created by having multiple solid buttons next to eachother.
+
+  ### SIZES
+    Button sizes convey heirarchy and meaning and can help drive attention and engagement to important calls-to-action. The default style should be uesd in the majority of cases. When grouping buttons be sure all buttons in the group are of the same size. Also keep in mind that the most extreme sizes are typically reserved for device specific circumstances (ie. extra-large buttons work well for the largest call to action on a desktop, while extra-small buttons are good for mobile.
+`
+
```

Comment:
Then Rest of these follows based on the Parent, As parent will be H2, then these children will be H3, H4 and so on..